OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -88,18 +110,27 @@ public override bool HandleMouseInput(MouseInput mi)
 					}
 					else if (dragStart.HasValue)
 					{
-						var newSelection = SelectActorsInBox(World, dragStart.Value, xy, _ => true);
+						var newSelection = DeadzoneSelectBox(World, dragStart.Value, xy, _ => true);
 						World.Selection.Combine(World, newSelection, mi.Modifiers.HasModifier(Modifiers.Shift), dragStart == xy);
 					}
+				} else
```

Comment:
Style nit: `else` should live on its own line, and should be combined with the `if`:
```
}
else if (useClassicMouseStyle)
	ApplyOrders(World, xy, mi);
```